tower-http/src/lib.rs

```diff
@@ -217,13 +218,16 @@
     clippy::match_like_matches_macro,
     clippy::type_complexity
 )]
-#![forbid(unsafe_code)]
+#![cfg_attr(not(test), forbid(unsafe_code))]
```

Comment:
Seems like all unsafe usage went away in later commits, so this can be reverted?